homeassistant/components/bring/todo.py

```diff
@@ -164,3 +190,20 @@ async def async_delete_todo_items(self, uids: list[str]) -> None:
                 raise HomeAssistantError("Unable to delete todo item for bring") from e
 
         await self.coordinator.async_refresh()
+
+    async def async_send_push_notification(
+        self,
+        notification_type: BringNotificationType,
+        item_name: str | None = None,
+    ) -> None:
+        """Send a push notification to members of the To-Do list."""
+        try:
```

Comment:
Should we check here whether an item_name for the notification type is required before sending it to the bring api?